### PR TITLE
Let GitHub build and publish a prebuilt release when a tag is push

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,9 @@ package:
 	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.$(HASH).tar.xz zld
 	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.dSYM.$(HASH).tar.xz zld.dSYM
 
+github_release: build
+	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.tar.xz zld
```
Comment (michaeleisel) [inline Makefile:41]: we've historically used .zip and not .tar.xz, any issue with .zip?

**Comment by michaeleisel** [comment]
very cool. is it possible to test beforehand, or do we just see for the next release?

File: Makefile (hunk 1 above)
Comment (thii) [reply Makefile:41]: No issue. `make package` uses `.tar.xz`, I just copy that.

Comment (thii) [reply Makefile:41]: Actually `zld.tar.xz` is about half the size of `zld.zip`, so any reason not using that?

**Comment by thii** [comment]
I'll test it in my fork.

File: Makefile (hunk 1 above)
Comment (michaeleisel) [reply Makefile:41]: tar is fine

**Comment by thii** [comment]
Needed to create a release first, it works now; verified here: https://github.com/thii/zld/runs/1628577401?check_suite_focus=true

**Comment by michaeleisel** [comment]
Thanks!

**Comment by thii** [comment]
Can you create and push a new tag?

**Comment by michaeleisel** [comment]
1.2.1 is already a fat binary though?

**Pull request merged**